app/src/main/java/com/example/funcy_portfolio_android/ui/login/LoginFragment.kt

```diff
@@ -32,7 +33,12 @@ class LoginFragment : Fragment() {
         }
 
         binding.btLogin.setOnClickListener {
-            findNavController().navigate(R.id.action_LoginFragment_to_MainFragment)
+            val mailAddress = binding.etMailAddress.text.toString()
+            if( mailAddress.matches(Regex("[a-zA-Z0-9._-]+@fun\\.ac\\.jp$"))) {
```

Comment:
現時点の対象ユーザが「学部」「博士前期（修士）」「博士後期（博士）」にしているから@以前の正規表現を学部の「b」，修士の「g」に限定しても良いと思う！（博士後期に関しては無視になってるけど...）

```suggestion
           if( mailAddress.matches(Regex("[b|g][0-9._-]{7}+@fun\\.ac\\.jp$"))) {
```

Reply:
あ，そうなんだ，対象ユーザに先生含まれてると思って雑な正規表現にしてしまった…
反映します！

-------
一応，いつか使う正規表現として完璧じゃないと思うけど，教員リストを見てそれっぽく作ったの置いておきます
[教員リスト（HOPE）](https://hope.fun.ac.jp/course/view.php?id=1114)
`(([a-z]{1,}-?[a-z])|([b|g][0-9._-]{7}))+@fun\\.ac\\.jp$`